### ISSUE TITLE
Add learning rate to observed hyperparameters in tensorboard

### ISSUE THREAD

**Issue by knasti** [knasti/PopNet#24]
Add learning rate to observed hyperparameters in tensorboard
